.eslintrc.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
```

Comment:
This file should be in the gitignore since beeil generates it.

Reply:
added to .gitignore